.idea/.gitignore

```diff
@@ -1,8 +0,0 @@
-# Default ignored files
-/shelf/
-/workspace.xml
-# Editor-based HTTP Client requests
-/httpRequests/
-# Datasource local storage ignored files
-/dataSources/
```

Comment:
exclude the .idea files also please
remove them first and commit, then add to git ignore